roles/edpm_ovn/tasks/run.yml

```diff
@@ -27,21 +27,36 @@
           - "{{ edpm_ovn_controller_tls_cacert_bundle_src }}:{{ edpm_ovn_controller_tls_cacert_bundle_dest }}:ro,z"
       when: cacert_bundle_exists.stat.exists
 
-- name: Ensure /usr/libexec/edpm-start-podman-container exists
-  ansible.builtin.import_role:
-    name: edpm_container_manage
-    tasks_from: shutdown.yml
-
 - name: Update ovn_controller health check script
   ansible.builtin.include_tasks:
     file: healthchecks.yml
 
-- name: Run ovn_controller container
+- name: Check for standalone container
+  ansible.builtin.stat:
+    path: "/etc/systemd/system/{{ item }}_ovn_controller.service"
+  register: standalone_service_file
+  loop:
+    - "edpm"
+    - "tripleo"
```

Comment:
what i'd like to understand though is why this is necessary. I can see that it is, but shouldn't the adoption job have already run edpm_tripleo_cleanup? If so, did it not cleanup?